googleapiclient/errors.py

```diff
@@ -37,6 +37,9 @@ class HttpError(Error):
   @util.positional(3)
   def __init__(self, resp, content, uri=None):
     self.resp = resp
+    if not isinstance(content, bytes):
+        # When see this exception in test, testdata may be wrong
```

Comment:
Drop this comment; test data passing through this function should be all corrected in this commit.